backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You back up applications by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[`Backup`] custom resource (CR).
+You back up applications by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[`Backup`] custom resource (CR) using OADP.
```

Comment:
For accessibility reasons, the link text does not equate to the target link heading (Backup > Creating a Backup CR). Using inline link text can become difficult for this reasons. Also, the text renders in monospace instead of the hyperlink formatting. 

Suggestion:

> You back up applications by creating a `Backup` custom resource (CR) by using OADP. See xref[Backing up and restoring applications].

Additionally, "using" should be "by using"

![Screenshot from 2023-06-08 10-00-26](https://github.com/openshift/openshift-docs/assets/57954076/9fb77b5a-8dc1-46ce-a392-4e98e2d7223e)

Reply:
Fixed.